opentracing-api/src/main/java/io/opentracing/ThreadLocalScheduler.java

```diff
@@ -0,0 +1,70 @@
+package io.opentracing;
+
+/**
+ * ThreadLocalScheduler is a trivial Scheduler implementation that relies on Java's thread-local storage primitives.
+ *
+ * @see Scheduler
+ * @see Tracer#scheduler()
+ */
+public class ThreadLocalScheduler implements Scheduler {
```

Comment:
I want to force all Tracers to always have some sort of `Scheduler` available, so it made sense (to me) to put this basic TLS one into `opentracing-api`. That said, it doesn't literally need to be.